### PR TITLE
docs: document arrow endpoint api

### LINKED ISSUE #1181: Update docs for feat: add an option to turn on compression for arrow output
A document change request is generated from https://github.com/GreptimeTeam/greptimedb/pull/4730

### PR DESCRIPTION
**Pull request by sunng87**
## What's Changed in this PR

Fixes #1181 

This patch includes arrow ipc http output format and others.

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/http.md [hunk 1]
```diff
@@ -152,6 +152,24 @@ curl -X POST \
     - `rows`：查询结果的行数据，每行是一个数组，包含 schema 中对应列的值。
 - `execution_time_ms`：该语句的执行时间，以毫秒为单位。
 
+### 其他输出格式
+
+### Alternative formats
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/http.md:157]:
```suggestion
```

File: docs/user-guide/protocols/http.md [hunk 2]
```diff
@@ -151,6 +151,24 @@ The response contains the following fields:
...
+  compatible format. Additional parameters:
+  - `epoch`: `[ns,u,µ,ms,s,m,h]`, returns epoch timestamps with the specified
+    precision
+- `csv`: output in comma separated values
+- `arrow`: [Arrow IPC
+  format](https://arrow.apache.org/docs/python/feather.html). Additional
+  parameters:
+  - `compression`: `zstd` or `lz4`, default: no compression
+- `table`: ASCII table format for console output
+
```
Comment (nicecui) [inline docs/user-guide/protocols/http.md:171]: Could you add an example?

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/http.md [hunk 3]
```diff
@@ -152,6 +152,43 @@ curl -X POST \
...
+
+```shell
+curl -X POST \
+  -H 'Authorization: Basic {{authorization if exists}}' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d "sql=SELECT * FROM monitor" \
+  http://localhost:4000/v1/sql?db=public&format=table
+```
+
+Output
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/http.md:181]:
```suggestion
结果：
```

**Review by nicecui — APPROVED** [review]
LGTM

**Review by fengjiachun — APPROVED** [review]
LGTM

**Pull request merged**